pkg/snapstore/swift_snapstore.go

```diff
@@ -396,6 +397,12 @@ func (s *SwiftSnapStore) Save(snap brtypes.Snapshot, rc io.ReadCloser) error {
 		return fmt.Errorf("failed uploading manifest for snapshot with error: %v", res.Err)
 	}
 	logrus.Info("Manifest object uploaded successfully.")
+
+	chunkDeleteCtx, chunkDeleteCancel := context.WithTimeout(context.TODO(), chunkUploadTimeout)
```

Comment:
```suggestion
	chunkDeleteCtx, chunkDeleteCancel := context.WithTimeout(context.Background(), chunkUploadTimeout)
```
It's better to use `context.Background()` if no parent context is passed to the function. Similar for other places where `context.TODO()` is used